server/app.js

```diff
@@ -1,11 +1,55 @@
+const { getRandomValues } = require('crypto')
```

Comment:
Are you able to replace this with an inbuilt function which returns a random value instead?